<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui</class>
 <widget class="QMainWindow" name="Gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gui Tests</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_7"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>638</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="ConsoleOutput">
   <property name="windowTitle">
    <string>Console Output</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="ConsoleOutputDock">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="AutoScroll">
           <property name="text">
            <string>Auto Scroll</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="NewestAtTop">
           <property name="text">
            <string>Newest At Top</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="MarkerA">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 0, 0);</string>
           </property>
           <property name="text">
            <string>Marker A</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="MarkerB">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 255, 0);</string>
           </property>
           <property name="text">
            <string>Marker B</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="MarkerC">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 0);</string>
           </property>
           <property name="text">
            <string>Marker C</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="MarkerD">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Marker D</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="ConsoleWindowView" name="ConsoleOutputTable"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionECIO_Outputs">
   <property name="text">
    <string>Enlightenment Monitor</string>
   </property>
  </action>
  <action name="actionECIO_Inputs">
   <property name="text">
    <string>Enlightenment Control</string>
   </property>
  </action>
  <action name="actionSHM_Control">
   <property name="text">
    <string>SHM Input</string>
   </property>
  </action>
  <action name="actionSHM_Output">
   <property name="text">
    <string>SHM Output</string>
   </property>
  </action>
  <zorder>ConsoleOutput</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConsoleWindowView</class>
   <extends>QTableView</extends>
   <header>gui.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
